_posts/2020/2020-01-16-what-working-with-a-distributed-team-taught-us-about-agile-team-collaboration.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: What working with a distributed team taught us about agile team collaboration
+date: 2020-01-16
+header_image: public/remote-work.jpg
+header_overlay: true
+category: methods-and-tools
+tags: ["scrum", "retro", "agile", "remote"]
+authors: ["Stefanie"]
+about_authors: ["sholler"]
+---
+
+
+One of our teams suddenly spread over four locations including onboarding two external developers.
+Before, it was a two location team, that met regularly for sprint change.
+In the beginning, one of our developers, who was already working remote for a longer time, gave me the following advice: never try to make things work the way they had been working for a co-located team, but to figure out alternative ways. 
```

Comment:
While reading through the post, I had problems with "co-located" and "collocated" - maybe we can find a way to make clearer what you refer to?